spec/models/group_spec.rb

```diff
@@ -197,18 +205,6 @@
     end
   end
 
-  def real_admins
```

Comment:
The name of this method says "real users", but the implementation is actually "human users".

"Real users" has a different, specific meaning.